external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -59,14 +59,17 @@ abstract class KafkaSinkSuiteBase extends QueryTest with SharedSQLContext with K
 
   protected def newTopic(): String = s"topic-${topicId.getAndIncrement()}"
 
-  protected def createKafkaReader(topic: String): DataFrame = {
-    spark.read
+  protected def createKafkaReader(topic: String, includeHeaders: Boolean = false): DataFrame = {
+    val df = spark.read
       .format("kafka")
       .option("kafka.bootstrap.servers", testUtils.brokerAddress)
       .option("startingOffsets", "earliest")
       .option("endingOffsets", "latest")
       .option("subscribe", topic)
-      .load()
+    if (includeHeaders) {
```

Comment:
nit: we can call `df.option("includeHeaders", includeHeaders.toString)` to save several lines here.